src/main/resources/native/android/c/launcher.c

```diff
@@ -208,3 +208,16 @@ void JVM_RawMonitorEnter() {
 void JVM_RawMonitorExit() {
     fprintf(stderr, "We should never reach here (JVM_RawMonitorExit)\n");
 }
+
+// AWT: Provide dummy methods for unresolved references to libawt
```

Comment:
Add the cause there (the text I have), since we need to remember where the U comes from so that we can remove it as soon as it is fixed in GraalVM.

Reply:
Ok, done